pkg/pgmodel/ingestor/trace/writer.go

```diff
@@ -126,12 +126,18 @@ func getServiceName(rSpan pdata.ResourceSpans) string {
 	return serviceName
 }
 
+var traceLabel = prometheus.Labels{"type": "trace"}
+var traceSpanLabel = prometheus.Labels{"type": "trace", "kind": "spans"}
```

Comment:
Lets keep the kind as per the convention in metric refactoring.

```suggestion
var traceSpanLabel = prometheus.Labels{"type": "trace", "kind": "span"}
```

lets also change `tags` to `tag` somewhere down.